addon/components/single-event.js

```diff
@@ -1,32 +1,36 @@
-import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 import { isBlank, isEmpty } from '@ember/utils';
 import layout from '../templates/components/single-event';
 import moment from 'moment';
 
 export default Component.extend({
-  store: service(),
   intl: service(),
   router: service(),
-  layout,
-  event: null,
+  store: service(),
+
   classNames: ['single-event'],
+
   'data-test-single-event': true,
+  event: null,
+  layout,
 
-  taughtBy: computed('intl.locale', 'event.instructors', function(){
-    const instructors = this.get('event.instructors');
-    if (isEmpty(instructors)) {
-      return '';
+  taughtBy: computed('event.instructors', 'intl.locale', {
+    get() {
```

Comment:
We're not using this pattern of having a getter anywhere even though it prevents writing to a compute property I find it hard to read and have removed it. Can you change that here?